rwgame/DrawUI.cpp

```diff
@@ -102,19 +103,20 @@ void drawPlayerInfo(PlayerController* player, GameWorld* world,
 
     infoTextY += ui_textHeight;
 
-    {
+    if (player->getCharacter()->getCurrentState().health > ui_lowHealth
+        || (int)(world->getGameTime() * 4.f) % 2) {
```

Comment:
how about `fmod`?
I'm totally against float pointers in general because they easily become inprecise due to float precision.
However, while we have them it looks like an even worse idea to convert them to integer.